airbyte-integrations/connectors/source-s3/integration_tests/spec.json

```diff
@@ -132,12 +133,75 @@
                 "type": "integer"
               }
             }
+          },
+          {
+            "title": "json",
+            "description": "This connector utilises <a href=\"https://pandas.pydata.org/docs/reference/api/pandas.io.json.read_json.html\" target=\"_blank\">Pandas</a> for JSON parsing.",
+            "type": "object",
+            "properties": {
+              "filetype": {
+                "title": "Filetype",
+                "const": "json",
+                "type": "string"
+              },
+              "orient": {
+                "title": "Orient",
+                "description": "Indication of expected JSON string format. Compatible JSON strings can be produced by to_json() with a corresponding orient value. Allowed values are {'split','records','index', 'columns','values', 'table'}",
+                "default": "columns",
+                "type": "string"
+              },
+              "convert_dates": {
+                "title": "Convert Dates",
+                "description": "A list of columns to parse for dates; If True, then try to parse date-like columns, default is True.",
+                "default": true,
+                "anyOf": [
+                  { "type": "boolean" },
+                  { "type": "array", "items": { "type": "string" } }
+                ]
+              },
+              "keep_default_dates": {
+                "title": "Keep Default Dates",
+                "description": "If parsing dates (convert_dates is not False), then try to parse the default datelike columns. A column label is datelike if one of these conditions is verified:\n            * it ends with '_at',\n            * it ends with '_time',\n            * it begins with 'timestamp',\n            * it is 'modified',\n            * it is 'date'\n            ",
+                "default": true,
+                "type": "boolean"
+              },
+              "lines": {
+                "title": "Lines",
+                "description": "Read the file as a json object per line.",
+                "default": true,
```

Comment:
How does the parser behave if `false`? 
I did not find any test file with has a multiline JSON object.

Reply:
You're right, I'll test this and remove the parameter if it's not applicable. In that case, I'll make sure to update all mentions of JSON to JSONL in the docs.

Reply:
I added a test for this and it now supports multiline JSON. However, it reads the whole file in memory at once. I added a disclaimer in the docs.